data/remnant/remnant outfits.txt

```diff
@@ -357,6 +360,7 @@ outfit "Aeon Cell"
 	"outfit space" -160
 	"energy generation" 21.9
 	"heat generation" 31.5
+	"ion resistance" 1
```

Comment:
So an Aeon Cell plus one Crystal Capacitor is enough to completely counteract all ion storms? The worst an ion storm can get is dealing 1.5 ion damage per frame. Aeon Cell + Crystal Capacitor = 1.5 ion resistance. Putting that into perspective, the ion DPS of an Ion Cannon is 0.083, meaning you'd need 18 Ion Cannons just to MATCH the resistance of this combo. And looking at a stock Albatross, which has an Epoch Cell on top of the Aeon Cell and Crystal Capacitor, you'd need 24 Ion Cannons to match the resistance of the ship.

Reply:
Values I'd suggest:
* Crystal Capacitor: 0.01
* Millennium Cell: 0.025
* Epoch Cell: 0.085
* Aeon Cell: 0.14

Reply:
Those values look a little low to me, but sure, worth trying out. Making the change.

Reply:
These are basically the minimum values to look at. We can ratchet them up from here.